packages/stencilbook/src/stencil-storybook.tsx

```diff
@@ -23,7 +23,35 @@ function loadStory(imps: any) {
   };
 }
 
-type Layout = "desktop" | "tablet" | "mobile";
+function getStoryFromKey(key: string, stories: OrganisedStoryWithSubs) {
+  if (!key) return undefined;
+
+  const keys = decodeURIComponent(key).split("-");
+  const group = keys[0];
+  const parentTitle = keys[1];
+  const subKey = keys[2];
+
+  const s = stories?.[group]?.find(
+    (element) => element.story.title === parentTitle
+  );
+  const subStory = s.subs?.[subKey];
```

Comment:
```suggestion
  const subStory = s?.subs?.[subKey];
```
this would explode if a story wasn't found in the `.find` above